.github/workflows/pull-request-checks.yaml

```diff
@@ -129,6 +129,14 @@ jobs:
       - name: Install bison and flex
         run: "${{env.SCRIPT_DIR}}\\install-bison.bat"
 
+      - name: Install z3
+        run: |
+          Invoke-WebRequest "https://github.com/Z3Prover/z3/releases/download/z3-4.8.8/z3-4.8.8-x64-win.zip" -OutFile z3.zip
+          Expand-Archive z3.zip
+          $z3_path = "$(Get-Location)\z3\$((Get-ChildItem z3).Name)"
```

Comment:
Can you add an explanation for why you need the get-childitem.name here? IIRC this is because the path includes the z3 version number which we expect to change, but this is weird enough to deserve a comment here.

SHA: 1dc52122584b6df8ddfe742a6f69c30333cc96de